src/diffpy/labpdfproc/tests/test_functions.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import pytest
+
+from diffpy.labpdfproc.functions import Gridded_circle
+
+params1 = [
+    [[1, 3, 1, {(0.0, -1.0), (0.0, 0.0), (-1.0, 0.0), (1.0, 0.0), (0.0, 1.0)}, 5]],
```

Comment:
these are easier to read if they are a list of tuples, one for each test, then this tuple has two lists which are for the inputs and outputs.  So in this case it would be 
```
params1 = [
    ( [1, 3, 1], [{(...), (...), (...)}, 9] ),
    (...)
]
@pytest.mark.paramaterize("inputs, expected", params1)
def test_...():
    circle = Griddedcircle(inputs[0], inputs[1])
```
and so on.  This is much easier to read.

I may not have the syntax from `pytest.mark.paramatrize()` quite right so please look it up, but we would like the intent of hte tests to be easier to read in all your tests.